airbyte-webapp/src/hooks/services/useDestinationDefinition.tsx

```diff
@@ -14,4 +14,10 @@ const useDestinationDefinitionList = (): {
   });
 };
 
-export { useDestinationDefinitionList };
+const useDestinationDefinition = (id: string): DestinationDefinition => {
```

Comment:
The main idea of adding name and ids into CreditConsumptionByConnector is to not make a bunch of additional requests to backend and be able to get info from request or cache.

Are we sending here request for every such entity? Or is it picked from cache? I believe we retrieve these entities only to display icons.

We should make sure that we either:
- pick info from cache for icons if possible
- expand CreditConsumptionByConnector entity with icon, so we won't make bunch of unnecessary requests.